draft-ietf-quic-transport.md

```diff
@@ -3630,6 +3627,9 @@ The Version Negotiation packet does not include the Packet Number and Length
 fields present in other packets that use the long header form.  Consequently,
 a Version Negotiation packet consumes an entire UDP datagram.
 
+A server MUST NOT send more than one Version Negotiation packet in response to a
+single UDP datagram.
```

Comment:
I believe this paragraph is the one you intend to add, and it looks fine to me.

OTOH I see lots of unrelated changes that essentially reverts the recently merged changes. Maybe you need to cherry-pick?